.travis.yml

```diff
@@ -155,8 +155,12 @@ matrix:
       env: TEST_SUITE=postgres DB=postgres TOTAL_TEST_GROUPS=2 TEST_GROUP=1 MIGRATIONS_TEST_MIGRATE=1
 
     - <<: *acceptance_default
-      name: 'Acceptance'
-      env: TEST_SUITE=acceptance USE_SNUBA=1
+      name: 'Acceptance (1/2)'
+      env: TEST_SUITE=acceptance USE_SNUBA=1 TOTAL_TEST_GROUPS=2 TEST_GROUP=0 PERCY_PARALLEL_NONCE=$(date +%s) PERCY_PARALLEL_TOTAL=2
```

Comment:
Not a bad idea to use `date`... is +%s seconds? This is going to be tricky to sync, since jobs can launch at different times. It might work better to use minutes, but I'm still not sure if that'll be stable